static/js/user_profile.js

```diff
@@ -32,8 +32,11 @@ function compare_by_name(a, b) {
 }
 
 function format_user_stream_list_item(stream, user) {
+    const is_under_my_access =
```

Comment:
Let's extract this computation as a new `people.can_admin_user(user)` function defined just after `get_bot_owner_user` in `people.js`, and just write:
```
    const show_unsubscribe_button =
        people.can_admin_user(user) || settings_data.user_can_unsubscribe_other_users();

```